forte/processors/data_augment/algorithms/text_replacement_op.py

```diff
@@ -37,11 +38,15 @@ def __init__(self, configs: Config):
         self.configs = configs
 
     @abstractmethod
-    def replace(self, input: Entry) -> str:
+    def replace(self, input: Entry) -> Tuple[bool, str]:
         r"""
         Most data augmentation algorithms can be considered
         as replacement-based methods on different levels.
         This function takes in an entry as input and
-        returns the augmented string.
+        Args:
+            - input: An entry contains the input text.
+        Returns:
```

Comment:
You should write:

Returns: a tuple, where the first is xxxx, and the second element is xxxx